_posts/0000-01-02-Brettfastone.md.

```diff
@@ -0,0 +1,6 @@
+---
+layout: slde
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I'm going to *Assume* that this is *Italic* and this is **Bold** and will be **EXTREMELY *HAPPY* if** I'm correct.